src/bundle/Resources/public/js/CKEditor/custom-attributes/ui/custom-attributes-form-view.js

```diff
@@ -209,9 +209,8 @@ class IbexaCustomAttributesFormView extends View {
         addListToDropdown(labeledDropdown.fieldView, itemsList);
 
         this.listenTo(labeledDropdown.fieldView, 'execute', (event) => {
-            const value = config.multiple
-                ? [...new Set([...labeledDropdown.fieldView.element.value.split(','), event.source.value])].join(',')
-                : event.source.value;
+            const alreadySelectedValues = labeledDropdown.fieldView.element.value ? labeledDropdown.fieldView.element.value.split(',') : [];
```

Comment:
maybe we can use optional chaining?
```suggestion
            const alreadySelectedValues = labeledDropdown.fieldView.element.value.split?.(',') ?? [];
```